model/user.go

```diff
@@ -487,6 +487,12 @@ func IsValidUserRoles(userRoles string) bool {
 	return true
 }
 
+// Make sure you acually want to use this function. In context.go there are functions to check permissions
+// This function should not be used to check permissions.
+func (u *User) IsGuest() bool {
```

Comment:
To check if a user is a guest we check if the user have the role `system_guest` as part of his roles.

Reply:
:+1: